openhands/server/routes/files.py

```diff
@@ -42,7 +43,7 @@
 
 
 @app.get('/list-files')
-async def list_files(request: Request, path: str | None = None):
+async def list_files(request: Request, path: str | None = None) -> dict[str, Any]:
```

Comment:
```suggestion
async def list_files(request: Request, path: str | None = None) -> dict[str, str]:
```

Should this be `str`?